src/stores/messages/MessagesDataProviderStore.ts

```diff
@@ -129,70 +128,67 @@ export default class MessagesDataProviderStore {
 			FIFTEEN_SECONDS,
 		);
 
-		let message: EventMessage | undefined;
+		if (!this.searchChannelPrev || !this.searchChannelNext) return;
+
+		const startTimestamp = queryParams.startTimestamp;
+
+		let message: EventMessage | null = null;
 
-		if (this.searchChannelPrev && this.searchChannelNext) {
-			this.isLoadingAnchorMessage = true;
-			if (this.messagesStore.selectedMessageId) {
+		this.isLoadingMessageIds = true;
```

Comment:
i cant see it being set to false somewhere

Reply:
fixed: https://github.com/th2-net/th2-rpt-viewer/pull/344/commits/7d5805aa0a5b76d57f20cd85c6574a52713f895a